app/core/api/classrooms.js

```diff
@@ -64,18 +64,35 @@ module.exports = {
     }))
   },
 
-  fetchByOwner (ownerId, options = {}) {
+  async fetchByOwner (ownerId, options = {}) {
     let projectionString = ''
     if (Array.isArray(options.project)) {
       projectionString += `&project=${options.project.join(',')}`
     }
     if (options.includeShared) {
       projectionString += '&includeShared=true'
     }
-    return fetchJson(`/db/classroom?ownerID=${ownerId}${projectionString}`, {
+    const classrooms = await fetchJson(`/db/classroom?ownerID=${ownerId}${projectionString}`, {
       callOz: options.callOz,
       method: 'GET'
     })
+
+    if (classrooms.length > 0) {
+      const classroomLanguages = [...classrooms.reduce((acc, classroom) => {
```

Comment:
we should first add classrooms into our vue store and then add this data so that classrooms become visible